### DISCUSSION TITLE
Gets connected to Jupyter Lab instead of VSCode

### DISCUSSION THREAD

**Discussion by ne1114** [coder/code-server#4368] [Q&A] [answered]
Hello, 

I used this command to install -> which I got `v3.12.0`.
```
curl -fsSL https://code-server.dev/install.sh | sh
``` 
After configuring I changed `config.yaml` to following : 
```
bind-addr: 127.0.0.1:8080
auth: none
password: blabla
cert: false
```
And then I used ssh port forwarding as suggested in the documentation. After port forwarding, opening `localhost:8080` in Chrome browser pops up `Juyper-lab` instead of `VSCode`. 

Does anyone know what's going on?

**Comment by jsjoeio** [comment, marked as answer]
This is very very strange. Do you have any special environment variables you're using? cc @code-asher

**Reply by ne1114** [reply]
Thanks for the reply @jsjoeio! Actually, it turned out that my VM was hosting a jupyter lab on port 8080.Once I changed the port for the code-server, the problem was resolved. 😅

**Reply by lg-nineoneninethreenine** [reply]
Well, that was kinda expected... Make sure that your Jupyter processes run on a safe port so that code-server doesn't need any meddling with ports. code-serer should have shown an error that port 8080 is being used by another process. Remember, code-server has an error with Jupyter notebooks, so for the notebooks, you need to use Jupyter or Jupyter Lab.

**Reply by jsjoeio** [reply]
Haha wow, what a funny coincidence. Glad you got it resorted (and thanks @im-coder-lg for the protip!)
